dev-java/gradle/gradle-3.0.ebuild

```diff
@@ -11,34 +10,44 @@ SRC_URI="http://services.gradle.org/distributions/${P}-src.zip"
 HOMEPAGE="http://www.gradle.org/"
 LICENSE="Apache-2.0"
 SLOT="${PV}"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~x86 ~amd64"
```

Comment:
If they'd been in the wrong order to start with, I'd have let you off but what was the point of this? I know this is being picky but other devs will pick you up on this too. Let's keep the noise down. :smiley_cat: